trade-finance-manager-ui/server/controllers/case/index.js

```diff
@@ -400,7 +424,8 @@ const getCaseFacility = async (req, res) => {
       facilityId,
       facilityTfm: facility.tfm,
       user: req.session.user,
-      showAmendmentButton: showAmendmentButton(deal, req.session.user.teams) && !hasAmendmentInProgress && !amendment.amendmentId,
+      showAmendmentButton:
+        showAmendmentButton(deal, req.session.user.teams) && !hasAmendmentInProgress && !amendment.amendmentId && !futureEffectiveDatePortalAmendment,
```

Comment:
```suggestion
        showAmendmentButton(deal, teams) && !hasAmendmentInProgress && !amendment.amendmentId && !futureEffectiveDatePortalAmendment,
```

Reply:
Do we need `!amendment.amendmentId` since we are already checking `!hasAmendmentInProgress`.